# Patch to remove long double suffix from some constants so the library can compile in POWER architectures.
# This does not change the results obtained with the library, as long doubles are not used anywhere else.
# See https://gitlab.com/libxc/libxc/commit/de20fdea12819f3ff90aad71a0e67475c29cce51
--- libxc-3.0.1/src/util.h.orig	2018-09-19 14:59:59.606624308 +0200
+++ libxc-3.0.1/src/util.h	2018-09-19 15:07:46.559366209 +0200
@@ -39,15 +39,15 @@
 float  asinhf(float  x);
 #endif
 
-#define M_SQRTPI        1.772453850905516027298167483341145182798L
-#define M_SQRT3         1.732050807568877293527446341505872366943L
-#define M_CBRT2         1.259921049894873164767210607278228350570L
-#define M_CBRT3         1.442249570307408382321638310780109588392L
-#define M_CBRT4         1.587401051968199474751705639272308260391L
-#define M_CBRT5         1.709975946676696989353108872543860109868L
-#define M_CBRT6         1.817120592832139658891211756327260502428L
-#define M_CBRT7         1.912931182772389101199116839548760282862L
-#define M_CBRT9         2.080083823051904114530056824357885386338L
+#define M_SQRTPI        1.772453850905516027298167483341145182798
+#define M_SQRT3         1.732050807568877293527446341505872366943
+#define M_CBRT2         1.259921049894873164767210607278228350570
+#define M_CBRT3         1.442249570307408382321638310780109588392
+#define M_CBRT4         1.587401051968199474751705639272308260391
+#define M_CBRT5         1.709975946676696989353108872543860109868
+#define M_CBRT6         1.817120592832139658891211756327260502428
+#define M_CBRT7         1.912931182772389101199116839548760282862
+#define M_CBRT9         2.080083823051904114530056824357885386338
 
 /* Very useful macros */
 #define min(x,y)  ((x<y) ? (x) : (y))
